### DISCUSSION TITLE
Is it possible to install the intel gpu plugin on a Talos Kube cluster ?

### DISCUSSION THREAD

**Discussion by SirSirae** [siderolabs/talos#5786] [Q&A]
Hello !

Before switching to Talos I would like to ask if anyone has tried to install the Intel GPU device plugin ([doc here](https://intel.github.io/intel-device-plugins-for-kubernetes/cmd/gpu_plugin/README.html)) and manage to use the hardware acceleration it provides ? 

I saw that nvidia GPU is in alpha but I have seen nothing about Intel GPU (and iGPU)

Thanks

**Comment by frezbo** [comment]
I believe folks at k8s-at-home discord had some success with this. Would you mind jumping to the talos-community slack and there should be someone to help with this.

**Reply by SirSirae** [reply]
I will ask there then thank you !
